tests/report.py

```diff
@@ -42,9 +50,14 @@ def add_case(testobj):
             report[scenario]["description"] = spec["description"]
 
     timestamp = datetime.datetime.now().strftime('%Y%m%d_%H%M%S')
-    filename = f"tests/output/report_{timestamp}.json"
-    with open(filename, "w") as file:
+    filename = f"report_{timestamp}.json"
+    fullpath = os.path.join("tests/output", "filename")
```

Comment:
should this be `integration-dashboard/knowledge-middleware/output/{report}`? I feel like `tests/output` is somewhat vague and we'll be using this bucket for testing for other TAs as well

Reply:
we're writing it locally to tests/output/report.py but we actually writing to `ta1/report.py` on S3